src/vault.py

```diff
@@ -198,7 +198,7 @@ def __init__(self, args, envs):
 
         # Setup Vault client (hvac)
         try:
-            self.client = hvac.Client(url=self.envs.vault_addr, token=os.environ["VAULT_TOKEN"])
+            self.client = hvac.Client(url=self.envs.vault_addr, namespace=os.environ["VAULT_NAMESPACE"], token=os.environ["VAULT_TOKEN"])
```

Comment:
It appears VAULT_NAMESPACE becomes mandatory when set this way, can it be made optional, and an additional error message added?